src/main/java/com/spotinst/sdkjava/model/repo/admin/account/SpotAccountAdminRepo.java

```diff
@@ -90,4 +92,22 @@ public RepoGenericResponse<Boolean> delete(String identifier, String authToken,
         return retVal;
     }
 
+    @Override
+    // The base 'IRepository' class getAll() method requires 3 parameters but this implementation needs only
```

Comment:
Make responseStatus as last parameter.

Reply:
how we are able to use the getAll method from the IRepository?

Reply:
Done